### ISSUE TITLE
More User Information

### ISSUE THREAD

**Issue by mongomay69** [dularion/streama#304]
I've been  using this for a little while and I haven't had any problems. Some things i would love to see implemented would be some more User options (as admin ofc).

1. The ability to see who is logged in
2. The ability to see what someone has been watching
3. The ability to clear user passwords (have them set them on login)
4. When you make a synchronized session see who else is currently watching with you
5. Building on what someone else said about restricting things to certain people you could make ranks and assign them to shows/movies and then give users those ranks to control what they see.

Just some idea's

**Comment by dularion** [comment]
I totally agree with the above! Im also interested in developing a feature similar to steam, like "invite to join" and "jump into game with.." but with videos :) This is definitely part of the roadmap!
